File: modulo1/segmento3/apunte7Funciones/ejercicio4.py
def par(num1):
	
	if num1%2 == 0:
		respuesta= True
	else:
		respuesta= False
	
	return respuesta

num1=int(input("Ingrese un numero: " ))

respuesta= par(num1)
if respuesta:
	print("El numero ingresado es par")
else:
	print("El numero ingresado es impar")
		

File: modulo1/segmento3/apunte7Funciones/Ejercicio4.java
import java.util.Scanner;

public class Ejercicio4{

	public static Boolean par(Integer num){
		Boolean respuesta=true;

		if(num%2==0){
			respuesta=true;
		}else{
			respuesta=false;
		}
		
		return respuesta;
	
	
	}
	public static void main(String args[]){
		Scanner teclado = new Scanner(System.in);
		Boolean respuesta;

		System.out.println("Ingrese un numero: ");
		Integer num= Integer.parseInt(teclado.nextLine());

		respuesta= par(num);

		if(respuesta){
			System.out.println("El numero es par");
		}else{
			System.out.println("El numero es impar");
		}
	
	}

}
